src/shared/components/query/CaseSetSelector.tsx

```diff
@@ -68,7 +76,7 @@ export default class CaseSetSelector extends QueryStoreComponent<{}, {}>
 				<ReactSelect
 					value={this.store.selectedSampleListId}
 					options={this.caseSetOptions}
-					clearable={this.store.selectedSampleListId != this.store.defaultSelectedSampleListId}
+					filterOption={(opt: ReactSelectOptionWithName, filter)=>_.includes(opt.textLabel.toLowerCase(), filter)}
```

Comment:
I think it's hard to write unit tests with external libraries. Almost by definition it has to be an integration test because you cant reach inside it.